features/storage/kvstore/conf/kv_config.cpp

```diff
@@ -134,6 +134,23 @@ int _storage_config_FILESYSTEM_NO_RBP();
 int _storage_config_tdb_external_common();
 int _storage_config_filesystem_common();
 
+/**
+ * @brief If block device out of Mbed OS tree is to support, please overwrite this
+ *        function to provide it.
+ *
+ * @returns pointer to other block device.
+ */
+BlockDevice *get_other_blockdevice();
+
+/**
+ * @brief This function checks if given block device is flash type. If block device
+ *        out of Mbed OS tree is to support, please overwrite this function to add check
+ *        for it.
+ *
+ * @returns true for flash type or false for non-flash type.
+ */
+bool is_blockdevice_flash_type(BlockDevice *bd);
```

Comment:
There's no need for this function. One can call the BD's `get_erase_value` method. A non flash device would return a -1 value, telling that that this block device is incapable of erasing.

Reply:
@davidsaada Yes. BD's `get_erase_value method` can help check flash type or not. I've updated it and related config description.